lib/tidewave/mcp.ex

```diff
@@ -11,6 +11,9 @@ defmodule Tidewave.MCP do
 
   @impl true
   def init(_init_arg) do
+    # TODO: remove when we drop support for OTP 27
+    :ets.new(:tidewave_tools, [:set, :named_table])
```

Comment:
```suggestion
    :ets.new(:tidewave_tools, [:set, :named_table, read_concurrency: true])
```

Reply:
Btw. I would totally put that in `init_tools` so it stays together, but your call!